libff/algebra/curves/bls12_381/bls12_381_init.cpp

```diff
@@ -0,0 +1,634 @@
+#include <libff/algebra/curves/bls12_381/bls12_381_g1.hpp>
+#include <libff/algebra/curves/bls12_381/bls12_381_g2.hpp>
+#include <libff/algebra/curves/bls12_381/bls12_381_init.hpp>
+
+namespace libff
+{
+
+bigint<bls12_381_r_limbs> bls12_381_modulus_r; // (VV)
+bigint<bls12_381_q_limbs> bls12_381_modulus_q; // (VV)
+
+bls12_381_Fq bls12_381_coeff_b;
```

Comment:
We seem to have these globals AND as static values on the group classes.  I'm guessing this is not introduced in this change, but a long-standing duplication in all curves.  Might be worth adding an issue to settle on a single place.